README.md

```diff
@@ -132,6 +138,23 @@ _           | `host.py`                   | (to be removed) control a remote dev
 **others**  | `launch.sh`                 | shell script to start `pvm.py` with one _click_
 _           | `build_omxplayer.sh`                 | shell script to build `omxplayer` with one _click_
 
+## How to deploy your code when you testing in multiple rpis?
+Use the [deploy_code_to_rpi.sh](https://github.com/omarcostahamido/PVM/blob/AddCDscript/deploy_code_to_rpi.sh) script.
+But there are two prerequisites:
+
+One is the need to set up passwordless ssh access.
+For the passwordless ssh access please follow [here](https://danidudas.medium.com/how-to-connect-to-raspberry-pi-via-ssh-without-password-using-ssh-keys-3abd782688a).
+
+Second is the need to set the autostart.
+
+Once you're done, run the command on your computer.
+
+`bash deploy_code_to_rpi.sh`
+
+`cd PVM && git pull`
+
+`sudo reboot`
```

Comment:
restarting the control machine?

Reply:
I have thought about this question for a long time. I realized that our Raspberry Pi must eventually be set to autostart, so I wanted our test environment to be in an autostart environment as well, so after multiple options, I chose this reboot format.

Reply:
as per line 148 `autostart` is a requirement on the rpi devices for this `deploy_code_to_rpi.sh` script to work as intended. What I am not clear about is why are you restarting the control machine at this point.

Reply:
There are two ways we want to use the new code, one is to close the existing pvm program and restart it. The second one is to restart the rpi. The reason why we choose the second one here is that the second one is more in line with our actual scenario. 
And I found in the actual test, the Raspberry Pi reboot time is short, less than 1 minute, acceptable.

Reply:
@KaneBetter got it. As per my comment [here](https://github.com/omarcostahamido/PVM/pull/26#discussion_r1008914713) I had not looked well at `deploy_code_to_rpi.sh` and I thought these two lines were being executed on the control machine. Thank you.